plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/factories/ControlFlowGraphFactory.java

```diff
@@ -180,44 +234,26 @@ private static boolean isExitingFinallyBlock(ComplexNodeMapper cnMapper, Node no
 		return isExitingFinallyBlock;
 	}
 
-	private static final class CFEComparator implements Comparator<ControlFlowElement> {
-		@Override
-		public int compare(ControlFlowElement cfe1, ControlFlowElement cfe2) {
-			return cfe1.hashCode() - cfe2.hashCode();
-		}
-	}
-
-	private static class CNMapper implements ComplexNodeMapper {
-		final private Map<ControlFlowElement, ComplexNode> cnMap;
-
-		CNMapper(Map<ControlFlowElement, ComplexNode> cnMap) {
-			this.cnMap = cnMap;
-		}
-
-		@Override
-		public ComplexNode get(ControlFlowElement cfe) {
-			return cnMap.get(CFEMapper.map(cfe));
-		}
-
-		@Override
-		public Iterable<ComplexNode> getAll() {
-			return cnMap.values();
-		}
-	}
-
 	/** Prints detailed information of jump nodes */
 	private static String getJumpTokenDetailString(JumpToken jumpToken, Node jumpNode) {
 		String jNode = ASTUtils.getNodeDetailString(jumpNode);
 		String jmpStr = jNode + " >> " + jumpToken.toString();
 		return jmpStr;
 	}
 
-	/** Prints detailed information of control flow edges. Used for debugging purposes */
-	private static void printEdgeDetails(ControlFlowEdge e) {
-		String sNode = ASTUtils.getNodeDetailString(e.start);
-		String eNode = ASTUtils.getNodeDetailString(e.end);
-		String edgeStr = sNode + ":" + e.toString() + ":" + eNode;
-		System.out.println(edgeStr);
+	/** Prints detailed information of all control flow edges. Used for debugging purposes */
+	private static void printAllEdgeDetails(ComplexNodeMapper cnMapper) {
```

Comment:
maybe change method to create string, caller can decide what to do with it, instead of just forcing stdout.

Reply:
also this can be separate utility, no need to pollute this class API (even if private)